packages/remix/src/utils.ts

```diff
@@ -12,3 +17,39 @@ export function findConfig(dir: string, basename: string): string | undefined {
 
   return undefined;
 }
+
+export function isLayoutRoute(
+  routeId: string,
+  routes: Pick<ConfigRoute, 'id' | 'parentId'>[]
```

Comment:
Out of curiosity, is the `Pick` necessary? If `id` is not used, they what's the harm in just declaring `routes: ConfigRoute`?

Reply:
It's just cause TypeScript complains in the tests without all the necessary props, which aren't directly necessary for this function.